kratos/solving_strategies/strategies/explicit_solving_strategy.h

```diff
@@ -0,0 +1,719 @@
+    {
+        rOStream << Info();
+    }
+
+    /// Print object's data.
+    virtual void PrintData(std::ostream &rOStream) const
+    {
+        rOStream << Info();
+    }
+
+    ///@}
+
+protected:
+    ///@name Protected static Member Variables
+    ///@{
+
+    // Level of echo for the solving strategy
+    int mEchoLevel;
+
+    // Settings for the rebuilding of the DOF set
+    int mRebuildLevel;
+
+    ///@}
+    ///@name Protected member Variables
+    ///@{
+
+    ///@}
+    ///@name Protected Operators
+    ///@{
+
+    ///@}
+    ///@name Protected Operations
+    ///@{
+
+    /**
+     * @brief Calculate the explicit update
+     * This method is intended to implement the explicit update calculation
+     * Note that it has to be implemented according to the explicit scheme in a derived class
+     */
+    virtual void SolveWithLumpedMassMatrix()
+    {
+        KRATOS_ERROR << "Calling the base ExplicitSolvingStrategy SolveWithLumpedMassMatrix(). Implement the specific explicit scheme solution update in a derived class" << std::endl;
+    }
+
+    /**
+     * @brief Get the Delta Time object
+     * This method returns the DELTA_TIME from the ProcessInfo container
+     * @return const double
+     */
+    virtual inline double GetDeltaTime()
+    {
+        return GetModelPart().GetProcessInfo().GetValue(DELTA_TIME);
+    }
+
+    ///@}
+    ///@name Protected  Access
+    ///@{
+
+    ///@}
+    ///@name Protected Inquiry
+    ///@{
+
+    ///@}
+    ///@name Protected LifeCycle
+    ///@{
+
+private:
+    ///@}
+    ///@name Static Member Variables
+    ///@{
+
+    ///@}
+    ///@name Member Variables
+    ///@{
+
+    ModelPart &mrModelPart;
+
+    bool mMoveMeshFlag;
+
+    bool mInitializeWasPerformed = false;
+
+    bool mInitializeSolutionStepWasPerformed = false;
+
+    ExplicitBuilderAndSolverPointerType mpExplicitBuilderAndSolver = nullptr;
+
+    ///@}
+    ///@name Private Operators
+    ///@{
+
+    ///@}
+    ///@name Private Operations
+    ///@{
+
+    /**
+     * @brief Initialize the DOF set values
+     * This method initializes the current value of the unknown variables in the DOF set
+     */
+    void InitializeDofSetValues()
+    {
+        // Initialize the DOF values
+        auto& r_dof_set = mpExplicitBuilderAndSolver->GetDofSet();
+        IndexPartition<int>(r_dof_set.size()).for_each(
+            [&](int i_dof){
+                auto it_dof = r_dof_set.begin() + i_dof;
+                auto &r_value = it_dof->GetSolutionStepValue();
+                r_value = 0.0;
+            }
+        );
+    }
+
+    /**
+     * @brief Auxiliary call to the Initialize()
+     * For a given container, this calls the Initialize() method
+     * @tparam TContainerType Container type template (e.g. elements, conditions, ...)
+     * @param rContainer Reference to the container
+     */
+    template <class TContainerType>
+    void InitializeContainer(TContainerType &rContainer)
```

Comment:
this is duplicated from the `Scheme`
Another reason to use `Scheme`

Reply:
I think that this is already clarified. Right? @philbucher